packages/web-components/src/components/ic-tab/ic-tab.css

```diff
@@ -68,3 +68,13 @@
 ::slotted(svg) {
   fill: var(--label-color);
 }
+
+@media (forced-colors: active) {
+  :host {
+    border-bottom: none;
+  }
+
+  :host(:not([selected])) {
+    height: 44px
+  }
+}
```

Comment:
Just noticed that the text on the tabs is misaligned slightly vertically - so they jump a bit when you select different tabs
![image](https://user-images.githubusercontent.com/111882109/205289061-43992e54-d0d9-4ff3-ab74-97b5c34674f0.png)